lnwire/features.go

```diff
@@ -102,6 +102,21 @@ var GlobalFeatures = map[FeatureBit]string{
 	StaticRemoteKeyRequired: "static-remote-key",
 }
 
+// Features is a mapping of known feature bits to a descriptive name. All known
+// feature bits must be assigned a name in this mapping, and feature bit pairs
+// must be assigned together for correct behavior.
+var Features = map[FeatureBit]string{
```

Comment:
the commits are structured this way so that each commit won't break existing behavior, so we first send flat features and then read them. the prior iteration did as you suggest but joost pointed out that the intermediate commits could break compatibility. in order to do accommodate, we have to first add `Features`, the later remove the local/global features

Reply:
yes, I found the pr better like this. But maybe if there isn't a perfect order, there are too many commits.

Reply:
Makes sense, agree on compatibility over reviewability for a small diff as this.